bookkeeper-server/src/main/java/org/apache/bookkeeper/server/http/service/TriggerGCService.java

```diff
@@ -62,6 +62,16 @@ public HttpServiceResponse handle(HttpServiceRequest request) throws Exception {
             response.setBody(jsonResponse);
             response.setCode(HttpServer.StatusCode.OK);
             return response;
+        } else if (HttpServer.Method.GET == request.getMethod()) {
+            boolean isInForceGC = bookieServer.getBookie().getLedgerStorage().isInForceGC();
+
+            String output = "Is Force Triggered GC on BookieServer: " + bookieServer.toString() + " running? "
```

Comment:
I would suggest creating an object called GarbageCollectionStatus, which includes 

- whether gc is running, and maybe at what phase (gc, minor compaction, major compaction)
- gc counters or maybe last gc time.

Bascially the idea is to provide a comprehensive view about what is going on within a bookie

Reply:
Good idea @sijie

But having such value as result of the  GET would not be consistent with a PUT as trigger.

Why don't have separate 'resources' ?

Having the status is very useful so +1

Reply:
@eolivelli fair enough. if having separate 'resources', what is your suggestion?

Reply:
Thanks, @sijie, Maybe @eolivelli means that start a new endpoint for the gc_details getting, and left this gc endpoint use for "force triggered gc".
I am fine with both way. do your suggestion in this endpoint, or start a new PR for gc_details

Reply:
gc_details sounds good.

Reply:
Ok for gc_details
I meant 'resources' in REST terms